velox/core/QueryConfig.h

```diff
@@ -111,6 +111,9 @@ class QueryConfig {
   static constexpr const char* kSpillFileSizeFactor =
       "spiller-file-size-factor";
 
+  static constexpr const char* kHashTableSizeIncreaseFactor =
+      "hash-table-size-increase-factor";
```

Comment:
Probably should be named `divider` instead of `factor`?

Reply:
Thanks for the suggestion!